### ISSUE TITLE
[EN][Raid] Atramedes

### ISSUE THREAD

**Issue by Hartwall** [Atlantiss/BugTracker#10107]
http://www.wowhead.com/spell=78353/roaring-flame-breath

**Description**: Describe the problem
Atramedes fire on flight stage doesnt have a proper move speed

**Current behaviour**: What happens
When atramedes starts flying, the fire catches the person in a second no matter how much the person moves.

**Expected behaviour**: What should happen instead
Atramedes fire should have a fixed movement speed

**Steps to reproduce the problem**:

1.  reach flight stage
2. use warlock portal, leap of faith on priest, stampeding roar and see how atramedes fire also gains ridicculous momentum
3. die

rev 5815

**Comment by xolon** [comment]
Flame should not have fixed speed, but it should start slowly and increase it's speed exponentially

**Comment by TranceAccount** [comment]
So the target not changing is wrong?

**Comment by xolon** [comment]
Yes, it's wrong, when someone uses the gong while Atramedes flames are chasing someone it should after the stun on the boss end switch on person that used the gong. https://youtu.be/LtG8bre4XlE?t=4m16s
